src/components/molecules/UserReactions/UserReactions.scss

```diff
@@ -2,7 +2,7 @@
 
 $reactionOffset: calc(-1 * #{$reactionSize} / 2);
 $shoutBounceTranslateXOffset: 50%;
-$reactionTimeout: var(--user-reactions-reaction-timeout, 5s);
```

Comment:
The value was always to 5s when the variable was used, which made this meaningless.